### ISSUE TITLE
Sensitivity analysis indicate unplausible relationship for dose

### ISSUE THREAD

**Issue by JanSchlender** [Open-Systems-Pharmacology/PK-Sim#848]
The sensitivity analysis for a compound which acts dose-linear should reveal zero sensitivity for dose with respect to PK parameters encountering dose in its calculation (e.g. CL=Dose/AUC). For such a compound AUC and AUC should increase or decrease in the same proportion which would yield to zero sensitivity. However, PK-Sim calculates a sensitivity of 1 for such cases. 

[Doselinearity.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/2080884/Doselinearity.zip)

**Comment by msevestre** [comment]
Probably CL is calculated using the original dose as oppose to the dose being varied.

**Comment by msevestre** [comment]
yeah that's the issue. This one is going to be a tough one to solve however

**Comment by msevestre** [comment]
The same also probably for any PK Parameter that depends on BW if some volume are varied. I am assuming that this is also off (xxx_norm)

**Comment by msevestre** [comment]
actually I think we should remove the output depending on input such as CL when input is being varied. This simply does not make sense

**Comment by Yuri05** [comment]
>we should remove the output depending on input such as CL when input is being varied. This simply does not make sense

:thinking:

**Comment by msevestre** [comment]
@Yuri05 Does that mean agree or disagree?

PKParameter CL depends in its calculation on the Dose itself so this is a double dependencies so to speak. What information can you win from that?

**Comment by Yuri05** [comment]
This means "Have to think about it". Need more opinions in any case.

**Comment by msevestre** [comment]
At any rate, at the moment this is wrong and implementation will be quite hard to implement because of the way we are caching some properties etc... easiest way is not to be wrong and remove PK-Parmeters sensitiivty for some parameters

**Comment by JanSchlender** [comment]
Just to get you right, you would remove e.g. dose from being available to the SA of CL or would you remove CL as an output of SA?

**Comment by msevestre** [comment]
>would you remove CL as an output of SA?
This would be the easiest obviously but not the most elegant solution

Instead I would suggest to remove in our case dose from the SA of CL. Choosing another PKParameters would still show dose however. 
The same should probably be done for all normalized parameter (AUC_norm is AUC / Dose/BW]. Probably some funky stuff can happen there too
